src/services/text_processor.py

```
@@ -1,39 +1,37 @@
 # src/services/text_processor.py
-import spacy
 import re
+import numpy as np
```

Ruff (0.12.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)